docs/external-node.md

```diff
@@ -334,6 +338,43 @@ please refer to the [getting-started guide](getting-started.md#open-vswitch).
       sudo systemctl start antrea-agent
       ```
 
+##### Container Install
+
+1. Build `antrea-agent` docker image in the root of the antrea code tree and copy the docker image
+   `projects.registry.vmware.com/antrea/antrea-ubuntu:latest` to the Linux VM.
+
+   ```bash
+   make
+   ```
+
+   Please follow the below steps to copy the `projects.registry.vmware.com/antrea/antrea-ubuntu:latest` to the Linux VM.
+
+   ```bash
+   # Rename the image
+   docker image tag <image-id> projects.registry.vmware.com/antrea/antrea-ubuntu:latest
+
+   # Save it in the tar file
+   docker save -o <tar file path in source host machine> projects.registry.vmware.com/antrea/antrea-ubuntu
+ 
+   # Copy this tar file using simple tools such as cp, rsync, scp, etc. or any other method that you prefer.
+   # Then load that image in the Linux VM.
+   docker load -i <path to image tar file>
+   ```
+
```

Comment:
The installation of docker should be handled by developer because they are the one copying the image file and loading that. So its their responsibility to install a docker on that machine.